src/styles/navigation.scss

```
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
```

according to my suggestion to rename the classes
```suggestion
  &__list {
```